pkg/odo/cli/service/service.go

```diff
@@ -54,9 +57,9 @@ A full list of service types that can be deployed are available using: 'odo cata
 		// make sure the service type exists
 		serviceType := args[0]
 		matchingService, err := svc.GetSvcByType(client, serviceType)
-		util.CheckError(err, "unable to create service because Service Catalog is not enabled in your cluster")
+		odoutil.CheckError(err, "unable to create service because Service Catalog is not enabled in your cluster")
 		if matchingService == nil {
-			fmt.Printf("Service %v doesn't exist\nRun 'odo service catalog' to see a list of supported services.\n", serviceType)
+			log.Errorf("Service %v doesn't exist. Use 'odo service catalog' to see a list of supported services.\n", serviceType)
```

Comment:
Why not break into two statements like in https://github.com/redhat-developer/odo/blob/88ec602332333ff561899ad06bb2a1a5109a7e5f/pkg/odo/cli/component/create.go#L146-L147 ?